src/abstraction/transient.rs

```diff
@@ -362,12 +362,15 @@ impl TransientKeyContext {
         label: Option<Data>,
     ) -> Result<PublicKeyRSA> {
         self.set_session_attrs()?;
-        let key_handle = self.context.context_load(key_context)?;
+        let key_handle = self
+            .context
+            .context_load(key_context)
+            .map(KeyHandle::from)?;
```

Comment:
Does it work if you just add an `into` at the end?
```rust
let key_handle = self.context.context_load(key_context)?.into();
```

Reply:
I am not sure if the compiler can figure out what convert into. I think rust requires you to be specific. The following would work (I think):
```rust
let key_handle: KeyHandle = self.context.context_load(key_context).into();
```
or 
```rust
let key_handle = self.context.context_load(key_context).into::<KeyHandle>();
```

Reply:
Yes that what I meant! You decide if you prefer that rather than calling `map`!